src/System.Console/src/System/Console.cs

```diff
@@ -696,6 +726,32 @@ public static void Write(String value)
             Out.Write(value);
         }
 
+        // TODO: Uncomment when ArgIterator and friends are available in System.Runtime.dll
```

Comment:
@KrzysztofCwalina, is there an issue open for exposing these types in System.Runtime.dll?